src/main/resources/net/rptools/maptool/language/i18n.properties

```diff
@@ -838,6 +838,127 @@ position.side.right         = Right
 position.side.top           = Top
 position.side.bottom        = Bottom
 
+# Start : CampaignPropertiesDialog light and sight.
+light.example.groupName = Light group name
+light.example.auras.groupName = Sample Auras: 5' grid
+light.example.name.aura.allPlayers = Aura All Players 10
+light.example.name.aura.donutCone = Donut Cone
+light.example.name.aura.donutHole = Donut Hole
+light.example.name.aura.gmRed = Aura GM Red 10
+light.example.name.aura.gmRedSquare = Aura GM Red Square
+light.example.name.aura.lineOfSight = Line of Sight
+light.example.name.aura.owner = Aura Owner 10
+light.example.name.aura.rangeArcs = Range Arcs
+light.example.name.aura.rangeCircles = Range Circles
+light.example.name.aura.sideFields = Side Fields
+light.example.name.forwardArcAura = Forward Arc
+light.example.name.lantern = Lantern
+sight.example.name.blind = Blind
+sight.example.name.coneVision = Conic Vision
+sight.example.name.darkVision = Dark Vision
+sight.example.name.elfVision = Elf Vision
+sight.example.comment = This line is a comment
+light.example.text.aura.allPlayers  = A blue aura visible to all players.
+light.example.text.aura.donutCone   = The same trick done with a cone.
+light.example.text.aura.donutHole   = This one hides a secret. If you have a range with no colour it becomes the range the light source begins. In this case the light begins at 20 and ends at 40.
+light.example.text.aura.gmRed       = A circular GM-only aura in red with a 15-foot diameter.
+light.example.text.aura.gmRedSquare = A 5-foot square red aura only visible to the GM.
+light.example.text.aura.lineOfSight = A beam 150 ft long with a width of 0.4 map units.
+light.example.text.aura.owner       = A circular green aura that is only visible to the token's owners.
+light.example.text.aura.rangeArcs   = A series of cones starting every 30 ft and ending 0.4 ft later.
+light.example.text.aura.rangeCircles= A series of circles starting every 30 ft and ending 0.4 ft later.
+light.example.text.aura.sideFields  = 4 cones with an arc of 90 degrees and range of 12.5, each one offset to cover a different region.
+
```

Comment:
Editorial only: Why have justified "equals" here, when this isn't done anywhere else?

Reply:
Residual effect of multiple edits and probably Notepad++ disagreeing with Spotless.